docs: add usage with ttypescript (#14)
wessberg/DI-compiler: README.md

program.emit(undefined, undefined, undefined, undefined, di({ program }));
```

### Usage with ttypescript

> To use DI-compiler With [`ttypescript`](https://github.com/cevek/ttypescript), you need to write a transformer and set the path to `tsconfig.json`.

Reviewer: ```suggestion
To use DI-compiler with [`ttypescript`](https://github.com/cevek/ttypescript), create a file that wraps the invocation of `di`:
```

```typescript:transformer.ts
import type { Program } from 'typescript'
import { di } from "@wessberg/di-compiler";

const transformer = (program: Program) => di({ program })

export default transformer
```

> 

Reviewer: ```suggestion
Then add a record to the `plugins` array of your `tsconfig.json` that maps a name you choose to the relative path to the file you jusit created:
```

```tsconfig.json
{
    "compilerOptions": {
        "plugins": [

>             { "transform": "path/to/transformer.ts" },

Reviewer: ```suggestion
            { "di": "path/to/my-di-transformer.ts" },
```

Author: It doesn't seem to work. `ttypescript`(@1.5.12) requires `"transform"` as key.

```
Error: tsconfig.json plugins error: transform must be present
```

Reviewer: > It doesn't seem to work. `ttypescript`(@1.5.12) requires `"transform"` as key.
> 
> ```
> Error: tsconfig.json plugins error: transform must be present
> ```

Ah, OK, I didn't know that. I'll remove the suggestion
